bin/install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+cd $DIR/..
+
+brew install php72 composer npm webpack wepback-dev-server
```

Comment:
imo global dependencies shouldn't be installed here.

Reply:
yea we can move the line to the headmen?